VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoUnbondMayaChain.swift

```diff
@@ -34,30 +37,44 @@ class TransactionMemoUnbondMayaChain: TransactionMemoAddressable, ObservableObje
             }
         }
     }
-    
+
     private var cancellables = Set<AnyCancellable>()
-    
-    required init(assets: [String]) {
+
+    let id = UUID()
```

Comment:
this one is not used anywhere either, could you please remove it?

Reply:
Removed